### PR TITLE
Feature/add backgound jobs

### PR DESCRIPTION
**Pull request by danypr92**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -179,6 +179,21 @@ ftp_password:                    # FTP password
...
+celery_password:                # Password to the user to talk with RabbitMQ
+celery_host:                    # Host of RabbitMQ server
+celery_port:                    # RabbitMQ port
+celery_vhost:                   # VHost RabbitMQ
+```
+
+* Flower variables
+```YAML
+flower_user=                    # User and password to access to the Flower UI
+Flower_password=                #
```
Comment (enricostano) [inline README.md]: `flower_password` and inline `#`

File: inventory/host_vars/local.tryton.coop/config.yml [hunk 2]
```diff
@@ -113,4 +113,14 @@ uwsgi_init_workers: 1
...
+celery_tryton_database: '{{ db_name }}'
+celery_user: 'eticom'
+celery_password: 'eticom'
+celery_host: 'localhost'
+celery_port: 5672
+celery_vhost: 'eticom'
 
+# Flower vars
+flower_user: 'flower'
+flower_password: 'flower_passwd'
```
Comment (enricostano) [inline inventory/host_vars/local.tryton.coop/config.yml]: not in secret?

File: inventory/host_vars/local.tryton.coop/config.yml [hunk 3]
```diff
@@ -113,4 +103,8 @@ uwsgi_init_workers: 1
 uwsgi_max_workers: '%k'
 uwsgi_listen_queue: 100
 
-
+# Celery Tryton
+celery_tryton_database: '{{ db_name }}'
+celery_host: 'localhost'
+celery_port: 5672
```
Comment (enricostano) [inline inventory/host_vars/local.tryton.coop/config.yml]: All other ports are in secrets, right?

**Comment by danypr92** [comment]
This PR is too old, we continue this work in #16

**Pull request closed without merging**